src/helm/snapteld.conf

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+# log_level for the snap daemon. Supported values are
+# 1 - Debug, 2 - Info, 3 - Warning, 4 - Error, 5 - Fatal.
+# Default value is 3.
+log_level: 3 
+
+# log_path sets the path for logs for the snap daemon. By
+# default snapteld prints all logs to stdout. Any provided
+# path will send snapteld logs to a file called snapteld.log in
+# the provided directory.
+# log_path: /snap
+
+# Gomaxprocs sets the number of cores to use on the system
+# for snapteld to use. Default for gomaxprocs is 1
+# gomaxprocs: 1
```

Comment:
Is single core enough to collect that much metrics? We may need to allow users to assign more cores, for example via chart config.